src/status_im/contexts/wallet/common/utils/networks.cljs

```diff
@@ -62,13 +62,13 @@
        :goerli-enabled?  goerli-enabled?}))))
 
 (defn network-list
-  [{:keys [balances-per-chain]} networks]
+  [chain-ids networks]
   (into #{}
         (mapv (fn [chain-id]
                 (first (filter #(or (= (:chain-id %) chain-id)
                                     (= (:related-chain-id %) chain-id))
                                networks)))
-              (keys balances-per-chain))))
+              chain-ids)))
```

Comment:
I know this code already existed, but there's a big room for improvement, for example: `first + filter` -> `some` and using the transducer syntax for `into` or just use `(set)` instead of `(into #{})`